samples/flutter_app/lib/main.dart

```diff
@@ -74,12 +77,40 @@ class _ChatWidgetState extends State<ChatWidget> {
   void initState() {
     super.initState();
     _model = GenerativeModel(
-        model: 'gemini-pro', apiKey: const String.fromEnvironment('api_key'));
+      model: 'gemini-pro',
+      apiKey: const String.fromEnvironment(
+        'api_key',
+      ),
+    );
     _chat = _model.startChat();
   }
 
   @override
   Widget build(BuildContext context) {
+    var textFieldDecoration = InputDecoration(
+      contentPadding: const EdgeInsets.symmetric(
+        vertical: 15,
+        horizontal: 15,
+      ),
```

Comment:
(nit) This is equivalent to:
```suggestion
      contentPadding: const EdgeInsets.all(15),
```